app/src/UserContext.ts

```diff
@@ -0,0 +1,9 @@
+import {createContext} from 'react';
+
+interface UserContextInterface{
+    username: string;
+}
+
+const UserContext = createContext<UserContextInterface | null>(null);
```

Comment:
```suggestion
const UserContext = createContext<UserContextInterface>(null);
```
don't need the extra `null` here